packages/core/theme-module/theme/components/AppHeader.vue

```diff
@@ -50,7 +50,8 @@ export default {
     };
     const { cart } = useCart();
     const cartTotalItems = computed(() => {
-      return getCartTotalItems(cart.value).toString();
```

Comment:
can't we handle it on UI?

Reply:
We can, but don't you think that even such micro-logic (if zero, make it null) doesn't fit in template?

Reply:
Issue reported to Storefront UI team: https://github.com/DivanteLtd/storefront-ui/issues/922